templates/opencraft-nginx.j2

```diff
@@ -30,6 +30,17 @@ server {
     access_log /var/log/nginx/{{ OPENCRAFT_DOMAIN_NAME }}_access.log;
     error_log /var/log/nginx/{{ OPENCRAFT_DOMAIN_NAME }}_error.log error;
 
+    # serve static files
+    #
+    location /static {
+        alias {{ OPENCRAFT_STATIC_FILES_DIR }};
+        try_files $uri $uri/ =404;
```

Comment:
What's the point of appending slashes for static assets?  Indexes are turned off, so I'm not sure what a path ending in a slash would do.  (I'm just curious what the use case is.  I'm not asking for this to be changed.)